tests/objects/generics/test_names.py

```diff
@@ -41,8 +41,8 @@ class TypeB:
         ([StrawberryUnion("Union", (TypeA, TypeB))], "UnionExample"),  # type: ignore
         ([TypeA], "TypeAExample"),
         ([CustomInt], "CustomIntExample"),
-        ([TypeA, TypeB], "TypeATypeBExample"),
-        ([TypeA, LazyType["TypeB", "test_names"]], "TypeATypeBExample"),  # type: ignore
+        ([TypeB], "TypeBExample"),
```

Comment:
I am uncertain what were these params, this is invalid generation, this class takes only one generic we should probably provide an error if someone tries this.